src/app/containers/Fragment/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { string, node, arrayOf } from 'prop-types';
+
+const italic = ({ children }) => <i>{children}</i>;
+const bold = ({ children }) => <b>{children}</b>;
+
+const attributeComponents = {
+  italic,
+  bold,
+};
+
+const Fragment = ({ text, attributes }) =>
+  /*
+    Iterates through the attribute array and returns a component based on the attribute type (i.e. 'italic' or 'bold').
+    These components are nested inside each other as children as the array is iterated through.
+    The text string is passed in as the initial value, so it is the first child or the returned value if there are no attributes.
+  */
+  attributes.reduce((previousAttribute, attribute) => {
+    const Attribute = attributeComponents[attribute];
```

Comment:
Prior to this, I think it would be worth explicitly returning null if the `attribute` does not exist in `attributeComponents`

Reply:
This would mitigate errors based on `Attribute` being undefined. An example would be passing `super` in as an attribute and seeing the error: `Warning: React.createElement: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: undefined...`

Reply:
Good thinking, I've added a fallback.